content/en/blog/_posts/2021-08-11-memory-qos-cgroups-v2.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: 'Support memory qos with cgroups v2'
```

Comment:
Maybe something like:
```suggestion
title: "Alpha in Kubernetes v1.22: quality-of-service for memory resources"
```
and a slug to match?

Reply:
Done